docs/guides/expectations/features_custom_expectations/how_to_add_example_cases_for_an_expectation.md

```diff
@@ -0,0 +1,152 @@
+To begin with, let's implement those two basic tests; one positive example case, and one negative example case. 
+
+#### 2. Defining our data
+
+Search for `examples = []` in the template file you are modifying for your new custom Expectation. 
+We're going to populate `examples` with a list of example cases.
+
+<details>
+  <summary>What is an example case?</summary>
+Each example is a dictionary with two keys:
+<ul>
+<li> <code>data</code>: defines the input data of the example as a table/dataframe. </li>
+<li> <code>tests</code>: a list of test cases that use the data defined above as input to validate against. </li>
+    <ul>
+        <li> <code>title</code>: a descriptive name for the test case. Make sure to have no spaces. </li>
+        <li> <code>include_in_gallery</code>: set it to True if you want this test case to be visible in the gallery as an example (true for most test cases). </li>
+        <li> <code>in</code>: contains exactly the parameters that you want to pass in to the Expectation. <inlineCode>{`"in": {"column": "x", "min_value": 4}`}</inlineCode> would be equivalent to <code>expect_column_max_to_be_between_custom(column="x", min_value=4)</code> </li>
+        <li> <code>out</code>: indicates the results the test requires from the <code>ValidationResult</code> needed to pass. </li>
+        <li> <code>exact_match_out</code>: if you set <code>exact_match_out=False</code>, then you don’t need to include all the elements of the result object - only the ones that are important to test, such as <inlineCode>{`{"success": True}`}</inlineCode>. </li>
+    </ul>
+</ul>
+</details>
+
+In our example, `data` will have two columns, "x" and "y", each with five rows. 
+If you define multiple columns, make sure that they have the same number of rows. 
+When possible, include test data and tests that includes null values (`None` in the Python test definition).
+
+```python file=../../../../tests/integration/docusaurus/expectations/creating_custom_expectations/expect_column_max_to_be_between_custom.py#L46
+```
+
+:::note
+When you define data in your examples, we will mostly guess the type of the columns. 
+Sometimes you need to specify the precise type of the columns for each backend. Then you use the `schema` attribute (on the same level as `data` and `tests` in the dictionary):
+
+```console
+"schemas": {
+  "spark": {
+    "x": "IntegerType",
+  },
+  "sqlite": {
+    "x": "INTEGER",
+  },
+```
+:::
+
+#### 3. Defining our tests
+
+In our example, `tests` will be a list containing dictionaries defining each test. 
+
+You will need to:
+1. Title your tests (`title`);
+2. Define the input for your tests (`in`);
+3. Decide how precisely you want to test the output of your tests (`exact_match_out`);
+4. Define the expected output for your tests (`out`).
```

Comment:
```suggestion
1. Title your tests (`title`).
2. Define the input for your tests (`in`).
3. Decide how precisely you want to test the output of your tests (`exact_match_out`).
4. Define the expected output for your tests (`out`).
```
Semicolon formatting of ordered lists is more formal than our style guide requires. In general, punctuation is not required for list items that are fragments, while sentences should start with a capital and end with a period. (And ideally a list will contain all sentences or all fragments as its items.)